openshift/system/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM centos/ruby-25-centos7
+FROM quay.io/centos7/ruby-25-centos7
```

Comment:
```suggestion
FROM registry.access.redhat.com/ubi7/ruby-27
```

Reply:
Then we will need to modify a bit the Dockerfile but yes

Reply:
Another option would be to start with clean UBI image and install `rh-ruby25` in build or is the package gone already?

Reply:
But not sure how UBI on RHEL7 works now. I think the following repos need to be enabled.

- rhel-7-server-rpms
- rhel-7-server-optional-rpms
- rhel-7-server-extras-rpms
- rhel-server-rhscl-7-rpms

Or maybe only the last one?

Reply:
We can't use UBI images or we should use 3rd party packages and then there is no point on using the UBI image anymore compared to centos